woocommerce/woocommerce-gutenberg-products-block
# Single Product: Fix the Align setting

This PR removes the unnecessary wrapper that's breaking the `align` setting.

## What

Fixes #10960 

## Why

Seems like this bug was introduced with this PR: https://github.com/woocommerce/woocommerce-blocks/pull/10374 - but I don't understand why we have the wrapper for the Single Product block in the first place.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. `Product Hero` pattern to a new page.
2. Change the `Align` to full.
3. Make sure the change is reflected on the front end.
4. Follow the testing steps from https://github.com/woocommerce/woocommerce-blocks/pull/10374 and make sure there's no regression. 


## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Single Product: Fix the Align setting.
